src/components/Chart/SeriesLabel.tsx

```diff
@@ -12,6 +12,7 @@ export interface SeriesLabelProps {
   color?: string
   value?: any
   outlierValue?: any
+  showLabel?: boolean
```

Comment:
```suggestion
  hideLabel?: boolean
```
So it follows the HTML convention of "absent = false"